src/node_url.cc

```diff
@@ -100,6 +100,11 @@ void BindingData::DomainToUnicode(const FunctionCallbackInfo<Value>& args) {
   CHECK(args[0]->IsString());
 
   std::string input = Utf8Value(env->isolate(), args[0]).ToString();
+  if (input.empty()) {
+    return args.GetReturnValue().Set(
+        String::NewFromUtf8(env->isolate(), "").ToLocalChecked());
```

Comment:
```suggestion
    return args.GetReturnValue().Set(FIXED_ONE_BYTE_STRING(env->isolate(), "")); 
```

Reply:
Can you apply the changes?

Reply:
@anonrig Oh, I'm so sorry. I checked this comment too late. I thought I fixed it, but maybe I didn't add a commit...
It seems that the PR has been merged, but I will fix it.